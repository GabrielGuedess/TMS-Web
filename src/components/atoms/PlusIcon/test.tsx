import { render } from '@testing-library/react';

import { PlusIcon } from '.';

describe('<PlusIcon />', () => {
  it('should render the PlusIcon correctly', () => {
    const { container } = render(<PlusIcon />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
