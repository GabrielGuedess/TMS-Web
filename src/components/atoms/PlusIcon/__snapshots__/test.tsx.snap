// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlusIcon /> should render the PlusIcon correctly 1`] = `
<svg
  fill="none"
  height="24"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M5 12h14m-7 7V5"
    stroke="currentColor"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
  />
</svg>
`;
